core/twig-extension.php

```diff
@@ -57,6 +58,26 @@ public function serviceLocationPathFilter($serviceLocationChannelId, $serviceId)
       return 'about:blank';
     }
     
+    public function pagePathFilter($page) {
+      $path = [];
+      
+      $currentPage = $page;
+      
+      while (true) {
+        if (empty($currentPage['parentId'])) {
+          break;
+        }
+        
+        $currentPage = \KuntaAPI\Pages\Loader::findOrganizationPage($currentPage['parentId']);
+        array_unshift($path, $currentPage['slug']);
+      }
```

Comment:
could cause management service to become slow if there is i.e. circular page - parent mapping error. failsafe max value for loop (i.e 1000)